content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md

```diff
@@ -7,6 +7,20 @@ menu:
     name: Release notes
 ---
 
+## v1.6.3 [2022-01-20]
+
+### Features
+
+- Support for the `attributes` field in Alerta node.
+- Add BigPanda handler options.
+- The `env var` config can now handle new items.
+- `Topic queue length` is now configurable.
```

Comment:
Is the `Topic queue length` related to the `DefaultEventBufferSize = 5000` and `MinimumEventBufferSize = 1000`?

Reply:
We need the config parameter names, default values and where it appears within the config file...

Reply:
Will need to update the config page - https://github.com/influxdata/docs-v2/issues/3710